### PR TITLE
added new & trending product section in home page

### PR DESCRIPTION
**Pull request by JantuDeb**
#16 
# Trending
- Trending tag in the product card
- Trending product list on home page

# New Arrivals
- Arrivals tag in the product card
- Arrivals product list on home page

image
image

### CODE REVIEW

File: src/components/products/Product.jsx [hunk 1]
```diff
@@ -28,9 +28,9 @@ const Product = ({ product }) => {
   return (
     <div className="card flex-col radius-md bg-white ">
       <div className="flex justify-between items-start">
-        <div className="text-white bg-red px-2 flex center radius-top-left">
-          Trending
-        </div>
+        {product?.tag && <div className="text-white bg-red px-2 flex center radius-top-left">
```
Comment (rkapoor10) [inline src/components/products/Product.jsx]: maybe you can change the colors

File: src/utils/utils.js [hunk 2]
```diff
@@ -1,2 +1,5 @@
 export const getDiscountedPrice = (price, discount) =>
   price - Math.ceil(discount * price);
+
+export const getProductWithTag = ({ tag, products }) =>
```
Comment (rkapoor10) [inline src/utils/utils.js:4]: Liked that you extracted out the function

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. Just go through the suggestions

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

**Pull request merged**